src/olympia/amo/decorators.py

```diff
@@ -181,22 +181,3 @@ def wrapper(request, *args, **kw):
         return response
 
     return wrapper
-
-
-def allow_mine(f):
```

Comment:
What does this have to do with this patch?

Reply:
Just a drive-by cleanup while I was looking at this file for stuff to change around auth